dev-setup.md

```diff
@@ -28,37 +27,31 @@ Run `make check-requisites` to check that all dependencies are installed.
 ## Development
 
 1. Run `make dependencies` to download the Go libraries needed to compile the project.
+1. Get a working development Kubernetes cluster. You can use:
 
-2. Get a working development Kubernetes cluster. You can use:
-
-    [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/#install-minikube)
+* [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/#install-minikube)
 
-      ```bash
-      make bootstrap-minikube
-      # Sets up a Minikube cluster with required resources
-      ```
+  ```bash
+  make bootstrap-minikube
+  ```
 
-    [Kind](https://kind.sigs.k8s.io/)
+* [Kind](https://kind.sigs.k8s.io/)
 
-      ```bash
-      make bootstrap-kind
-      # Sets up a kind cluster with required resources
-      ```
+  ```bash
+  make bootstrap-kind
+  ```
 
-    [GKE](https://cloud.google.com/kubernetes-engine/)
+* Cloud providers
 
-      Make sure that container registry authentication is correctly configured as described [here](https://cloud.google.com/container-registry/docs/advanced-authentication). When installing the `gcloud` tool, make sure to install the beta components by running `gcloud components install beta`.
+  Use [deployer](/hack/deployer/README.md) (note that some [one time configuration](/hack/deployer/README.md#typical-usage) is required):
 
-      ```bash
-      export GCLOUD_PROJECT=my-project-id
-      make bootstrap-cloud
-      # Sets up GKE cluster (by default) with required resources
-      ```
+  * [GKE](https://cloud.google.com/kubernetes-engine/): `make switch-gke bootstrap-cloud`
+  * [AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/): `make switch-aks bootstrap-cloud`
```

Comment:
It reminds me that there is a `switch-ocp` , should we document it somewhere ?

Reply:
I did consider that, but I think it needs additional discussion. I won't block this PR on that.

Reply:
Maybe create an issue to discuss how Openshift should be documented ?